app/assets/javascripts/envelopes.js.coffee.erb

```diff
@@ -359,3 +359,10 @@ $ ->
       else
         icon.addClass('glyphicon-expand').removeClass('glyphicon-collapse-down')
 
+  $('#populate-suggestions').click (event) ->
+    event.preventDefault()
+    leafEnvelopes = $('#fill_envelopes .amount.input').not('.sum')
+    leafEnvelopes.each (i, env) ->
+      $env = $(env)
+      toFund = $env.parent().find('.to-fund')[0].innerHTML.trim()
```

Comment:
I feel like this is the more jQuery way of doing it:
`toFund = $env.parent().find('.to-fund').html().trim()`

There are many functions like `html()` that operate on the first item found, so you don't need the `[0]`.

Reply:
Actually, to be safe, it would probably be better to use `.text()` to make sure that if there is any HTML added to that in the future, we'll only grab the text.